docs/configuration/command_line_interface.rst

```diff
@@ -71,104 +71,122 @@ These are the commands you may need to use in relation to your Mautic instance.
 .. vale off
 
 .. list-table:: 
-   :widths: 100 100
+   :widths: 25 50 25
    :header-rows: 1
 
    * - Command
      - Description
-   * - mautic:assets:generate
+     - Aliases
+   * - ``mautic:assets:generate``
      - Combines and minifies Asset files (CSS/JS) from each bundle into single production files
-   * - mautic:broadcasts:send
+     - 
+   * - ``mautic:broadcasts:send``
      - Process Contacts pending to receive a Channel broadcast.
-   * - mautic:campaigns:execute
+     - 
+   * - ``mautic:campaigns:execute``
      - Execute specific scheduled events.
-   * - mautic:campaigns:messagequeue
-     - Process sending of messages queue, you can also use the alias of ``mautic:messages:send``.
-   * - mautic:campaigns:messages
-     - Process sending of messages queue, you can also use the alias of ``mautic:messages:send``.
-   * - mautic:campaigns:rebuild
-     - Rebuild Campaigns based on Contact Segments, you can also use the alias of ``mautic:messages:update``.
-   * - mautic:campaigns:trigger
+     - 
+   * - ``mautic:campaigns:rebuild``
+     - Rebuild Campaigns based on Contact Segments.
+     - ``mautic:messages:update``
+   * - ``mautic:campaigns:trigger``
      - Trigger timed events for published Campaigns.
-   * - mautic:campaigns:update
-     - Rebuild Campaigns based on Contact Segments, you can also use the alias of ``mautic:messages:rebuild``.
-   * - mautic:campaigns:validate
+     - 
+   * - ``mautic:campaigns:validate``
      - Validate if a Contact has been inactive for a decision and execute events if so.
-   * - mautic:citrix:sync
+     - 
+   * - ``mautic:citrix:sync``
      - Synchronizes registrant information from Citrix products
-   * - mautic:cache:clear
+     - 
+   * - ``mautic:cache:clear``
      - Clears Mautic cache, by using this command, you will erase the 10-minute Mautic cache, which contains things like segment counts and data for dashboard widgets.
-   * - mautic:contacts:deduplicate
+     - 
+   * - ``mautic:contacts:deduplicate``
      - Merge Contacts based on same unique identifiers
-   * - mautic:custom-field:create-column
+     - 
+   * - ``mautic:custom-field:create-column``
      - Creates the actual column in the table
-   * - mautic:email:fetch	
-     - Fetch and process monitored Email, you can also use the alias of ``mautic:email:fetch``.
-   * - mautic:emails:send
+     - 
+   * - ``mautic:email:fetch``
+     - Fetch and process monitored Email.
+     - ``mautic:email:fetch``
```

Comment:
This seems to be a duplication in the alias column (the same as the first column).